django_prometheus/db/common.py

```diff
@@ -35,6 +38,31 @@ def __exit__(self, typ, value, traceback):
             self._counter.labels(**self._labels).inc()
 
 
+class DurationMeasurer(object):
```

Comment:
There is no need for this since the Histogram metric class already implements a context manager:
https://github.com/prometheus/client_python#histogram

Reply:
https://github.com/prometheus/client_python/blob/049744296d216e6be65dc8f3d44650310f39c384/prometheus_client/metrics.py#L560